terraform/files/template/cluster-template.yaml

```diff
@@ -114,7 +118,7 @@ spec:
           RandomizedDelaySec=15m
 
           [Install]
-          WantedBy=multi-user.target
+          WantedBy=timers.target
```

Comment:
The changes work as expected, but the `etcd-defrag.service` is not started (only enabled), hence the defragmentation does not work until the control plane node is rebooted. 

Steps to reproduce:
1. Create a management node and cluster
```
OS_CLOUD=test1 make create
```
2. SSH to the control plane node
3. Check if the timer is scheduled
```
# systemctl list-timers --all
NEXT                        LEFT         LAST                        PASSED              UNIT                         ACTIVATES                     
Wed 2023-03-08 11:40:05 UTC 2h 8min left Wed 2023-02-01 11:38:10 UTC 1 months 4 days ago motd-news.timer              motd-news.service             
...         
n/a                         n/a          n/a                         n/a                 etcd-defrag.timer            etcd-defrag.service 
```

I do not know if the above is intended. If not this could be fixed e.g. by `systemctl enable --now` (in postKubeadmCommands)